### ISSUE TITLE
Is this draft aborted?

### ISSUE THREAD

**Issue by MrJia1997** [wcsmith/draft-quic-receive-ts#3]
Hi there, I wonder if this draft is aborted, since the current revision has expired. If yes, then what's the reason?

**Comment by ianswett** [comment]
This is being used by a Google internal project, but it's not actively being worked on besides that.  Christian also has a timestamp draft that has not been adopted, and it's unclear to me what the working group wants to do about timestamps.

Is this something you're using or would like to use?

**Comment by MrJia1997** [comment]
Yeah. I'm trying to optimize CC algorithm for QUIC, and one-way delay measurement will enable me a better queueing and jitter detection. I've checked out Christian's draft too, but, IMHO, the encoding efficiency of TIMESTAMP frame doesn't look really good, since only one receive timestamp is carried per ack frame. Though handling reordered packets might be tough in this draft, it looks more promising for me to encode packet timestamps in a much more compact way.

**Comment by ianswett** [comment]
This draft now has a new author, and there's renewed interest, so activity should resume and hopefully we'll get it adopted by the QUIC WG in the near future.
